ssz/sedes/serializable.py

```diff
@@ -27,11 +34,13 @@
 TSerializable = TypeVar("TSerializable", bound="BaseSerializable")
 
 
-class MetaBase:
-    fields = None
-    field_names = None
-    field_attrs = None
-    container_sedes = None
+class Meta(NamedTuple):
+
+    has_fields: bool
+    fields: Optional[Tuple[Tuple[str, BaseSedes]]]
+    container_sedes: Optional[Container]
+    field_names: Optional[Tuple[str]]
```

Comment:
`Tuple[str]` denotes a length-1 tuple with a single string.   Should probably be `Tuple[str, ...]` for all of these.